helpers/date-parser.js

```diff
@@ -0,0 +1,52 @@
+/* eslint-disable camelcase */
+const months = [
```

Comment:
Why migrating from Moment.js?

Reply:
use a library weighing 50kb in order to render 1 paragraph, in my opinion, this is not reasonable, and react-moment used only on blog and post pages, so in other pages where we really need moment it still stays

Reply:
OK, makes sense.

Regarding code itself, I'd suggest you to use nice snippets from https://github.com/you-dont-need/You-Dont-Need-Momentjs#string--date-format